provider_sources/xyzservices-providers.json

```diff
@@ -373,5 +373,98 @@
             "max_zoom": 12,
             "name": "MapTiler.Terrain"
         }
+    },
+    "OrdnanceSurvey": {
+        "Road_3857": {
+            "url": "https://api.os.uk/maps/raster/v1/zxy/Road_3857/{z}/{x}/{y}.png?key={key}",
+            "html_attribution": "Contains OS data &copy Crown copyright and database right {year}",
+            "attribution": "Contains OS data (C) Crown copyright and database right {year}",
```

Comment:
I've now also noticed the `{year}` placeholder here. We currently don't support placeholders outside of the `url` parameter, so you will need to spell it out here. The attribution here shall always have the current year? We may need to figure out some autoupdating logic within https://github.com/geopandas/xyzservices/blob/main/provider_sources/_compress_providers.py 

I can probably take care of it within the follow-up PR alognside tests.